Sources/NIOPosix/BaseSocketChannel.swift

```diff
@@ -1218,6 +1253,10 @@ class BaseSocketChannel<SocketType: BaseSocketProtocol>: SelectableChannel, Chan
         }
     }
 
+    public final func connect0(to address: SocketAddress, promise: EventLoopPromise<Void>?) {
```

Comment:
total nit, to make the diff easier to read, I'd move this implementation up and the new one down :)

Reply:
Done.